wiki/Announcements/Showcases/en.md

```diff
@@ -1,29 +1,27 @@
-Showcases
-=============
+# Showcases
 
-Showcases
-----------
+Showcases on various topics help show the osu!community 
```

Comment:
I have no idea what happened there..
I'm going with `Showcases on various topics in the osu!community.`